packages/grid/_modules_/grid/utils/getGridLocalization.ts

```diff
@@ -2,13 +2,13 @@ import { GridLocaleText } from '../models/api/gridLocaleTextApi';
 import { GridOptions } from '../models/gridOptions';
 import { getMuiVersion } from './utils';
 
-export interface LocalizationV4 {
+interface LocalizationV4 {
   props: {
     MuiDataGrid: Pick<GridOptions, 'localeText'>;
   };
 }
 
-export interface LocalizationV5 {
```

Comment:
Nice catch. I bet there are even more places like this where functions or typings are exported even though they are not used anywhere else. We should keep an eye on that.